convert_json_to_txt.py

```diff
@@ -0,0 +1,76 @@
+import pandas as pd
+import os
+import cv2
+import json
+
+# This script converts VIA (VGG image Annotator) json files to .txt files required for YOLOv3.
+
+def create_txt_files(via_json_file, images_folder, txt_dest_folder, attribute_name, labels):
+	"""
+	Input:
+	via_json_file = json files exported from VIA annotator
+	images_folder = path folder containing images
+	txt_dest_folder = destination folder for txt files
+	label_idx_dictionary = dictionary whose keys are label names
+						   in VIA annotator and values are the 
+						   corresponding integers to be use for 
+						   YOLOv3. For example: 
+						   {'label0': 0, 'label1':1, 'label3': 3}
+	attribute_name = attribute name assigned from VIA annotator
+	Output:
+		Side effects: it creates .txt files whose names correspond
+					to the images names in train. 
+		None
+	"""
+	label_idx_dictionary = labels 
+	print(attribute_name)
+	data = json.load(open(via_json_file, "r"))
+	for name in list(data.keys()):
+		file_data = data[name]
+		regions = file_data["regions"]
+		image_name = file_data["filename"]
+		image_path = os.path.join(images_folder, image_name)
+		if not os.path.exists(image_path): 
+			print(f"[Warning] The image {image_name} does not exist.")
+			continue
+		image_height, image_width, _ = cv2.imread(image_path).shape
+		for region in regions:
+			if len(region)==0:
+				continue
+			f = open(os.path.join(txt_dest_folder, os.path.splitext(image_name)[0] + ".txt"), "w+")
+			if region["shape_attributes"]["name"] == "rect":
```

Comment:
Instead of:
```
if region["shape_attributes"]["name"] == "rect":
  ABC
else:
  XYZ
  continue
```
you can do:
```
if region["shape_attributes"]["name"] != "rect":
  XYZ
  continue
ABC
```